Gemfile.lock

```diff
@@ -132,7 +125,7 @@ GEM
     debug_inspector (0.0.3)
     deprecation (1.0.0)
       activesupport
-    devise (4.7.1)
+    devise (4.7.0)
```

Comment:
@cdmo devise version down back to 4.7.0 🤔

Reply:
🤦‍♂ oops

Reply:
```suggestion
    devise (4.7.1)
```